src/gettingstarted/first-project.md

```diff
@@ -58,12 +58,15 @@ You can also initialize your project using any of the pre-made templates below.
 
 <div>
   <h4>Examples</h4>
-  <a href="https://github.com/platformsh/platformsh-example-python">Generic Python application</a><br />
+  <a href="https://github.com/platformsh/platformsh-example-python-uwsgi">Python UWSGI with Unix Sockets</a><br />
+  <a href="https://github.com/platformsh/platformsh-example-python-elasticsearch">Python Elastic Search</a><br />
 </div>
 
 <div>
   <h4>Frameworks</h4>
   <a href="https://github.com/platformsh/platformsh-example-django">Django</a>
+  <a href="https://github.com/platformsh/platformsh-example-python-2.7">Platform.sh Python 2.7 Flask</a><br />
+  <a href="https://github.com/platformsh/platformsh-example-python-3.6">Platform.sh Python 3.6 Flask</a><br />
```

Comment:
@ajm348 If it's going to take a while to reformulate these let's leave them out for now.  The main fix here is not pointing to the deprecated repo, which we should get in sooner rather than later.